geocity/apps/permits/models.py

```diff
@@ -202,6 +212,43 @@ def __str__(self):
         return str(self.group)
 
 
+class DuoConfig(models.Model):
+    name = models.CharField(_("name"), max_length=128)
+    description = models.CharField(_("description"), blank=True, max_length=128)
+    is_active = models.BooleanField(
+        _("Actif"),
+        default=True,
+        help_text=_(
+            "Décocher afin de désactiver cette configuration. Les groupes ayant cette configuration, ne seront plus redirigés vers duo"
+        ),
+    )
+    client_id = models.CharField(
```

Comment:
Should thoses fields be encrypted like for KNOX Tokens ?
From @monodo on another branch